backend/middleware/authenticateToken.js

```diff
@@ -1,33 +1,27 @@
-const jwt = require("jsonwebtoken")
-
+const jwt = require("jsonwebtoken");
 
 const authenticateToken = (req, res, next) => {
-
-    // Get the token from the auth header (where it would be stored)
-    const authHeader = req.headers["authentication"];
-    const token = authHeader.split(' ')[1]
-
-    console.log(authHeader);
-
-    // Verify that there is a token
-    if(!token) {
-        // Bad request
-        // Code subject to change
-        res.sendStatus(401);
+  // Get the token from the auth header (where it would be stored)
+  const authHeader = req.headers["authentication"];
```

Comment:
Similar to the comment with the use of "authentication", `Authorization` should be used instead

Reply:
Fixed: Changes are reflected in recent commits